src/components/Clock.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+type Props = {
+  clockName: string,
+};
+
+type State = {
+  today: string,
+};
+
+export class Clock extends React.Component<Props, State> {
+  timerId = 0;
+
+  state: State = {
+    today: new Date().toUTCString().slice(-12, -4),
```

Comment:
You save here exactly the time, not the entire date
```suggestion
    currentTime: new Date().toUTCString().slice(-12, -4),
```